docs/development.md

```diff
@@ -98,43 +98,37 @@ This is what a rendering test looks like:
     );
     cy.get('svg');
   });
-  ```
-
-
-### **Docs or it didn't happen**
-
-Finally, if it is not in the documentation, no one will know about it and then **no one will use it**. Wouldn't that be sad? With all the effort that was put into the feature?
-
-The docs are located in the docs folder and are written in markdown. Just pick the right section and start typing. If you want to add to the structure as in adding a new section and new file you do that via the [sidebar](/.sidebar.md).
+```
 
-The changes in master is reflected in the  [Mermaid Docs](https://mermaid-js.github.io/mermaid/) once released,  the updates are commited to [https://mermaid-js.github.io/#/](https://mermaid-js.github.io/mermaid/#/).
+### Questions and/or suggestions?
```

Comment:
Could it be replaced by "Any question or suggestion ?" or at least replace "and/or" by "or". As the "or" is not exclusive, the intention of the sentence wouldn't be lost.

Reply:
Thanks. I submitted a new commit with the change 👍